mycore-iiif/src/main/java/org/mycore/iiif/image/resources/MCRIIIFImageResource.java

```diff
@@ -0,0 +1,172 @@
+/*
+ *  This file is part of ***  M y C o R e  ***
+ *  See http://www.mycore.de/ for details.
+ *
+ *  This program is free software; you can use it, redistribute it
+ *  and / or modify it under the terms of the GNU General Public License
+ *  (GPL) as published by the Free Software Foundation; either version 2
+ *  of the License or (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful, but
+ *  WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with this program, in a file called gpl.txt or license.txt.
+ *  If not, write to the Free Software Foundation Inc.,
+ *  59 Temple Place - Suite 330, Boston, MA  02111-1307 USA
+ *
+ */
+
+package org.mycore.iiif.image.resources;
+
+import java.awt.image.BufferedImage;
+import java.io.UnsupportedEncodingException;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.net.URLEncoder;
+
+import javax.imageio.ImageIO;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.Response;
+import javax.ws.rs.core.StreamingOutput;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.mycore.access.MCRAccessException;
+import org.mycore.iiif.image.impl.MCRIIIFImageImpl;
+import org.mycore.iiif.image.impl.MCRIIIFImageNotFoundException;
+import org.mycore.iiif.image.impl.MCRIIIFImageProvidingException;
+import org.mycore.iiif.image.impl.MCRIIIFUnsupportedFormatException;
+import org.mycore.iiif.image.model.MCRIIIFImageInformation;
+import org.mycore.iiif.image.model.MCRIIIFImageProfile;
+import org.mycore.iiif.image.model.MCRIIIFImageQuality;
+import org.mycore.iiif.image.model.MCRIIIFImageSourceRegion;
+import org.mycore.iiif.image.model.MCRIIIFImageTargetRotation;
+import org.mycore.iiif.image.model.MCRIIIFImageTargetSize;
+import org.mycore.iiif.image.parser.MCRIIIFRegionParser;
+import org.mycore.iiif.image.parser.MCRIIIFRotationParser;
+import org.mycore.iiif.image.parser.MCRIIIFScaleParser;
+import org.mycore.iiif.model.MCRIIIFBase;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+
+import static org.mycore.iiif.image.MCRIIIFImageUtil.*;
+
+@Path("/iiif/image")
+public class MCRIIIFImageResource {
+    public static final String IIIF_IMAGE_API_2_LEVEL2 = "http://iiif.io/api/image/2/level2.json";
+
+    private static final Logger LOGGER = LogManager.getLogger();
+
+    @GET
+    @Produces("application/ld+json")
+    @Path("{impl}/{identifier}/info.json")
```

Comment:
every method starts with `impl` path parameter. Should be path parameter of class